### DISCUSSION TITLE
AnyCPU Building error with WinForms

### DISCUSSION THREAD

**Discussion by SirIntellegence** [cefsharp/CefSharp#3461] [Q&A] [answered]
So, in the past, the CefSharp.WinForms package had a build directory with a `.props` and a `.target` file in it. The 88.2.90 version does not have it. I found this out because I have a program to update CEF binaries for a project without checking said binaries and such into version control. In the previous version, it didn't modify the project correctly, but I was able to manually fix the paths and it would work since the folder was there, but it isn't at the moment.
Was this intentionally removed for some reason? And, if so, what am I supposed to use instead to get "@(CefSharpWinFormsBinaries32)" and "@(CefSharpWinFormsBinaries64)"?

**Comment by amaitland** [comment, marked as answer]
> ⚠️ **Nuget Packages have been restructured** please make sure you read https://github.com/cefsharp/CefSharp/issues/3319 for details

As per the the [Release notes](https://github.com/cefsharp/CefSharp/releases/tag/v88.2.90) the Nuget packages underwent major changes. 

CefSharp.Winforms.dll is now a single AnyCpu dll. The targets and props files were removed as there is no need for them, the Nuget package now complies with the Nuget standard,  the dll is included in the lib folder and directly included in your project file. 

The nupkg file can be unzipped if you wish to inspect it's contents.

**Reply by SirIntellegence** [reply]
Ah, ok. Thanks!
